src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/PortableInfobox.java

```diff
@@ -225,177 +223,124 @@ public PortableInfobox open(String articleTitle) {
     return this;
   }
 
-  public PortableInfobox compareURLAndExternalLink(String externalLinkName,
-                                                   String externalNavigatedURL) {
-    Assertion.assertEquals(externalLinkName, externalNavigatedURL);
-
-    return this;
+  public String getHorizontalItemLabelFontSize(int index) {
+    return horizontalItemLabels.get(index).getCssValue("font-size");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "font-size" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)